.pre-commit-hooks.yaml

```diff
@@ -22,11 +22,22 @@
     - file
 
 -   id: validate-oas-schema
-    name: Validate schema
+    name: Validate yaml schema
     description: This hook validates a schema provided in yaml format
     entry: python -m dati_playground validate --validate-jsonschema=true
     language: python
-    files: '.*\.schema.yaml'
```

Comment:
Be aware that removing hook ids (or changing their defaults) potentially breaks the workflows of people using that particular hook.

Every hook already defined can be considered an exported symbol.